runner.go

```diff
@@ -115,26 +117,35 @@ func (r *Runner) Run() error {
 }
 
 func (r *Runner) syncLoop() {
+	ticker := time.NewTicker(r.fullResyncPeriod)
+	defer ticker.Stop()
 	for {
 		select {
 		case <-r.sync:
-			if !r.canSync {
-				log.Logger.Warn("Cannot sync peers while canSync flag is not set")
-				continue
-			}
-			err := r.syncPeers()
-			MetricsSyncPeerAttempt(r.device.Name(), err)
-			if err != nil {
-				log.Logger.Warn("Failed to sync wg peers", "err", err)
-				r.requeuePeersSync()
-			}
+			r.doSyncPeers()
+		case <-ticker.C:
+			log.Logger.Info("Full sync ticker expired, attempting a peers sync")
+			r.doSyncPeers()
 		case <-r.stop:
 			log.Logger.Debug("Stopping sync loop")
 			return
 		}
 	}
 }
 
+func (r *Runner) doSyncPeers() {
```

Comment:
"do"? ))))

Just `syncPeers`?

Reply:
`syncPeers` is actually implemented below, and this is calling it and handles errors. It was initially a part of the sync loop above but split in a new func to avoid duplication.

Reply:
Ok, but `doSyncPeers` is still not a good name, its basically `syncPeers2`.

If it doesn't return errors, something like `mustSyncPeers`?